tests/src/rules/no-unused-modules.js

```diff
@@ -663,7 +663,35 @@ describe('test behaviour for new file', () => {
   });
 });
 
-describe('test behaviour for new file', () => {
+describe('test behavior for destructured exports', () => {
+  beforeEach(() => {
+    fs.writeFileSync(testFilePath('./no-unused-modules/file-destructured.js'), '', { encoding: 'utf8' });
+  });
```

Comment:
my hope is to avoid any fs operations here - can these not be committed, with their contents?

Reply:
Uhh, sure just a moment, not sure why You prefer committed files though

Reply:
It's more deterministic, doesn't require any sort of permissions to run tests, and is much easier to debug.